native/templates/openharmony/entry/src/main/ets/MainAbility/cocos/game.ts

```diff
@@ -62,6 +62,8 @@ export function launchEngine (): Promise<void> {
         window.global = window;
         const systemReady = require('./jsb-adapter/sys-ability-polyfill.js');
         systemReady().then(() => {
+            // @ts-ignore
+            window.oh.loadModule = loadModule;
```

Comment:
Why not set the same logic as other platforms? Such as
```
window.require = loadModule;
```

Reply:
only oh need use loadModule. 
![image](https://user-images.githubusercontent.com/9692838/185024999-a0a420bb-c4e9-48b7-b5d3-b73ca63466b7.png)

Reply:
Does oh also has `window.require`?

Reply:
has , but require path need use locaModule convert.